src/tests/App.test.js

```diff
@@ -0,0 +1,27 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import data from './data';
+import App from '../App';
+
+const reducer = (
+  state = {
+    home: { home: data },
+  },
+) => state;
+
+const store = configureStore({ reducer });
+describe('App test', () => {
+  it('will test App rendering', () => {
+    render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <App />
+        </Provider>
+      </MemoryRouter>,
+    );
```

Comment:
- [ OPTIONAL ] When I run the tests, I get a console error as shown below, I suggest that you consider fixing it. It is not enough for the tests to pass, but as good developers, we also need to pay attention to console errors.

<img width="1021" alt="Screenshot 2023-07-01 at 22 43 03" src="https://github.com/momo-87/air-metrics-webapp/assets/64774936/1105787e-052d-4395-af18-55d6bef2f491">